app/GRAPHQL/components/SDLButton/SDLButton.tsx

```diff
@@ -0,0 +1,84 @@
+'use client';
+
+import { useTransition } from 'react';
+import styles from './SDLButton.module.css';
+import { createSDLQuery } from '@/lib/actions/form.actions';
+import Button from '@mui/material/Button';
+import { useTranslations } from 'next-intl';
+import { endpointSDLSchema } from '@/lib/formValidationSchema/validationSchema';
+import * as Yup from 'yup';
+
+type SDLExplorerButtonProps = {
+  open: boolean;
+  setOpen: (data: boolean) => void;
+  endpointURL: string;
+  endpointSDL: string;
+  setErrors: (data: Record<string, string>) => void;
+  errors: Record<string, string>;
+};
+
+const SDLButton = ({
+  open,
+  setOpen,
+  endpointURL,
+  endpointSDL,
+  setErrors,
+  errors,
+}: SDLExplorerButtonProps) => {
+  const t = useTranslations('apiClient');
+  const SDLSchema = endpointSDLSchema(t);
+
+  const [isPending, startTransition] = useTransition();
+
+  const handleClick = async () => {
+    if (endpointURL || endpointSDL) {
+      const valueSDL = endpointSDL ? endpointSDL : endpointURL;
+      const formData = {
+        valueSDL,
+      };
+      try {
+        await SDLSchema.validate(formData, { abortEarly: false });
+
+        setErrors({});
+        if (open) {
+          setOpen(false);
+        }
+
+        startTransition(async () => {
+          try {
+            const result = await createSDLQuery(`${valueSDL}`);
+            if (result && '__schema' in result) {
+              setOpen(true);
+            }
+          } catch (e) {}
```

Comment:
Can we handle this `catch` statement logic somehow?

Reply:
According to the technical specifications, if we click on the documentation button, but it does not exist, then nothing should happen